Create language about special Community Seat elections (#609)
istio/community: steering/CHARTER.md

        the seat for the term.
    1. Because the goal of Community Seats is to increase the perspectives on
    the Steering Committee, employees of a Company that holds Contribution

>     Seats are ineligible to hold a Community Seat.

Reviewer: Do we need to mention by when the special election will be held? Additionally, since we have 1 seat less until then how does the voting/decision process work?

Reviewer 2: We don't specify a timeframe for the regular elections, so I didn't want to put one here.
The seat is community voted, not Steering voted.

Reviewer: I was referring to voting within Steering for decisions with 1 seat less (i.e. 12 seats)  if the elections and filling the seat takes longer than what we desire.

Reviewer 2: Oh, sorry; the charter expresses all voting in terms of percentage of the Seats, and in the common case, 60% of 12 or 13 both round up to needing 8 votes. I don't see that causing any issues.

Author: I thought about the timing question. I started to write something, and then personally thought that boxing ourselves in was unnecessary. "Special Election" by definition means at an unusual time.

I don't think the number of seats in Steering matters, since this election is from the community. For other decisions when Steering is understaffed, the 60%/80% thresholds apply.

>         1. If a Community Seat thus becomes vacant due to the holding company
>         earning a Contribution Seat, a special election will be held in the
>         weeks following these events, in the same manner as described above
>         for the regular Community Seat elections.

Reviewer 2: ```suggestion
        1. If a Community Seat becomes vacant due to a Company earning a
        Contribution Seat while holding a Community Seat, a special election
        will be held, in the manner descried above for the annual Community
        Seat elections.
```

PTAL, tried to tidy the language a little.

Now we're making special elections a thing, should we also apply the same process to "If an individual changes company affiliation mid-term"?
